prism-highlighter.html

```diff
@@ -59,7 +59,12 @@
         return;
       }
 
+      if(event.detail._highlighted === true) {
```

Comment:
@43081j Would someone ever have two prism-highlighters under the same parent? Would love to understand that edge case a little more before merging this.

FWIW `_detectLang` doesn't need access to `this` internally, and could be a static function outside of the Prism module. But for now lets try to avoid refactoring that just to fix this issue.